test/clt-tests/core/test-unfreeze-defrosts.rec

```diff
@@ -62,3 +62,27 @@ sleep 5; mysql -h0 -P9306 -e "select count(*) from orders\G"
 ––– output –––
 *************************** 1. row ***************************
 count(*): 10000
+––– input –––
+mysql -h0 -P9306 -e "drop table if exists t; create table t; freeze t; flush ramchunk t;" &
+––– output –––
+[1] %{NUMBER}
+––– input –––
+sleep 1; searchd --stop
+––– output –––
++---------------------------------+---------------------------------+
+| file                            | normalized                      |
++---------------------------------+---------------------------------+
+| /var/lib/manticore/t/t.meta     | /var/lib/manticore/t/t.meta     |
+| /var/lib/manticore/t/t.ram      | /var/lib/manticore/t/t.ram      |
+| /var/lib/manticore/t/t.settings | /var/lib/manticore/t/t.settings |
++---------------------------------+---------------------------------+
```

Comment:
This input/output pair looks wrong. I believe if you increase the sleep value to some 5 seconds the output will be different which is not good as it makes the test depend on some randomness.